support-frontend/assets/components/thankYou/thankYouModule.tsx

```diff
@@ -93,14 +110,46 @@ const bodyContainer = css`
 	grid-area: body;
 `;
 
+const bodyAppsContainer = css`
+	> div {
+		display: flex;
+		justify-content: space-between;
+		margin-top: ${space[6]}px;
+	}
+`;
+
+const bodyTop = css`
+	border-bottom: 1px solid ${neutral[86]};
+`;
+const bodyStyle = css`
+	max-width: 240px;
+	${from.mobileMedium} {
+		max-width: 340px;
+	}
+`;
+
 const bodyCopyStyle = css`
 	${body.small()};
-	margin-top: ${space[3]}px;
 	margin-bottom: ${space[1]}px;
+	${from.tablet} {
+		font-size: 17px;
+	}
 
+	> h2 {
```

Comment:
Could we not just apply this CSS to the `h2`? This question goes for other uses of the child-combinator.

Reply:
Yep, can see this type of styling is difficult to follow, have removed this `> h2` and another `>  div` further down now too.